.travis.yml

```diff
@@ -28,9 +28,9 @@ matrix:
 install:
   - pip install tox
 script:
-  - export PATH=$(readlink -f ffmpeg-3.3.1-64bit-static):$PATH
+  - export PATH=$(readlink -f ffmpeg-3.3.4-64bit-static):$PATH
```

Comment:
It would be nice to not have to change this when the ffmpeg release version changes.

Would it make sense to use `-o ffmpeg.tar.xz` in the `curl` command so that the file gets saved to a particular filename?